src/parser.rs

```diff
@@ -4687,6 +4687,54 @@ impl<'a> Parser<'a> {
         Ok(idents)
     }
 
+    /// Parse identifiers of form ident1[.identN]*
+    pub fn parse_multipart_identifier(&mut self) -> Result<Vec<Ident>, ParserError> {
```

Comment:
Looking at this function more closely, I wonder how if we can't reuse `parse_identifiers` from above?

Or maybe we could add some more documentation (like docstrings / tests) for this function to make it clearer it is designed to parse strings into identifiers 🤔

Reply:
I think `parse_identifiers` above is the odd one out, as it accepts any delimiter between words/idents (except equal sign) which means it would parse `test + one + two` as a valid sequence of identifiers. This is why I introduced the new `parse_multipart_identifier` to be more strict about what constitutes a compound/multipart identifier.

Though I did base `parse_multipart_identifier` primarily on Spark SQL/Postgres syntax, so maybe `parse_identifiers` is generic to accommodate some other type of syntax

Reply:
Upon some more reflection, I think we should accept this code and update the documentation strings on `parse_multipart_identifier` and  `parse_identifiers` to explain in more detail what they do and how they are different.

I can try and find time to update the documentation maybe next week -- or @Jefffrey  do you have time to do so?

Reply:
I've update the doc a bit, let me know any further suggestions